wrap/python/gtsam_py/circlePose3.py

```diff
@@ -0,0 +1,40 @@
+from math import pi, cos, sin
+import numpy as np
+
+import gtsam
+from gtsam.util import Point3
+
+
+def circlePose3(numPoses=8, radius=1.0, symbolChar=0):
```

Comment:
I still see *two* circlePoses, but maybe you did not get to this yet. Also, why gtsam_py and not gtsam for the dicectory name?

Reply:
It is left as is from duy's naming convention. I believe he is doing this because of some conflicts in target name, but I can try changing it to `gtsam`.